install.sh

```diff
@@ -4,10 +4,17 @@
 
 set -e
 
+case $(uname -s) in
+Darwin) os="osx" ;;
+*) os="linux" ;;
+esac
+
 if [ $# -eq 0 ]; then
-	version="v0.2.7"
+	deno_uri=$(curl -sS https://api.github.com/repos/denoland/deno/releases?per_page=2 |
+		grep -o "https://github.com/denoland/deno/releases/download/v[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}/deno_${os}_x64\.gz" |
```

Comment:
Probably `"https://github.com/denoland/deno/releases/download/\.*\.gz"` works

Reply:
Almost. Without the backslash it seems to work fine.

```diff
+"https://github.com/denoland/deno/releases/download/.*\.gz"
-"https://github.com/denoland/deno/releases/download/\.*\.gz"
```

Reply:
Actually, no, that doesn't work - It will take the `linux` version on `osx` then.

Will use: `"https://github.com/denoland/deno/releases/download/.*/deno_${os}_x64\.gz"`

Reply:
> Probably `"https://github.com/denoland/deno/releases/download/\.*\.gz"` works

But why? Why lose all meaning? 😞